tests/dbt/cloud/hooks/test_dbt_hooks.py

```diff
@@ -0,0 +1,168 @@
+
+    @pytest.mark.asyncio
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_job_details")
+    async def test_get_job_status(self, mock_get_job_details):
+        """Test get_adf_pipeline_run_status function with mocked status"""
+        mock_get_job_details.return_value = SAMPLE_RESPONSE
+        hook = DbtCloudHookAsync(dbt_cloud_conn_id="test_conn")
+        response = await hook.get_job_status(self.RUN_ID, self.ACCOUNT_ID)
+        assert response == 1
+
+    @pytest.mark.asyncio
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_job_details")
+    async def test_get_job_status_exception(self, mock_get_job_details):
+        """Test get_adf_pipeline_run_status function with mocked status"""
+        mock_get_job_details.side_effect = AirflowException("Bad request")
+        hook = DbtCloudHookAsync(dbt_cloud_conn_id="test_conn")
+        with pytest.raises(AirflowException):
+            await hook.get_job_status(self.RUN_ID, self.ACCOUNT_ID)
+
+    @pytest.mark.asyncio
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_headers")
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_request_url_params")
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_connection")
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.aiohttp.ClientSession.get")
+    async def test_get_job_details(
+        self, mock_get, mock_get_conn, mock_get_request_url_params, mock_get_headers
+    ):
+        mock_get_conn.return_value = Connection(
+            conn_id=self.CONN_ID,
+            conn_type="test",
+            login=1234,
+            password="newT0k3n",
+            schema="Tenant",
+            extra=json.dumps(
+                {
+                    "login": "test",
+                    "password": "newT0k3n",
+                    "schema": "Tenant",
+                }
+            ),
+        )
+        mock_get_headers.return_value = HEADERS
+        mock_get_request_url_params.return_value = "/test/airflow/", {}
+        mock_get.return_value.__aenter__.return_value.json = AsyncMock(return_value=SAMPLE_RESPONSE)
+        hook = DbtCloudHookAsync(dbt_cloud_conn_id=self.CONN_ID)
+        response = await hook.get_job_details(self.RUN_ID)
+        assert response == SAMPLE_RESPONSE
+
+    @pytest.mark.asyncio
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_headers")
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_request_url_params")
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.DbtCloudHookAsync.get_connection")
+    @mock.patch("astronomer.providers.dbt.cloud.hooks.dbt.aiohttp.ClientSession.get")
+    async def test_get_job_details_without_account_id_exception(
+        self, mock_get, mock_get_conn, mock_get_request_url_params, mock_get_headers
+    ):
+        mock_get_conn.return_value = Connection(
```

Comment:
no docstring present, and check for all other test as well

Reply:
Fixed